src/widgets/survey/model/operations.ts

```diff
@@ -53,6 +54,32 @@ export const getItemIds = (pipeline: PipelineItem[]): string[] => {
   );
 };
 
+export const fillNullsForHiddenItems = (
+  itemIds: string[],
+  answers: AnswerDto[],
+  context: Array<{ itemId: string; isHidden: boolean; type: string }>,
+): { answers: AnswerDto[]; itemIds: string[] } => {
+  const modifiedAnswers: Array<AnswerDto> = [];
+  const modifiedContext = context.filter(i => i.type !== 'Splash');
```

Comment:
Please slightly modify (input paramer change to type: ActivityItemType) and re-use existing function canItemHaveAnswer

Reply:
modifiedContext -> filteredOriginalItems